### PR TITLE
Renames ModelUri to Storageuri

### LINKED ISSUE #281: Rename modelUri/model-initializer to storageUri/storage-initializer
#222 added the concept of storageUri

As soon as we have v1alpha2, rename:
* modelUri -> storageUri
* model-initializer -> storage-initializer

### PR DESCRIPTION
**Pull request by rakelkar**
Fixes #281 

This change is mostly search/replace - but does require a new container to get built (side-car container name has changed)

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/327)

### CODE REVIEW

**Comment by rakelkar** [comment]
Verified that kfserving continues to work after this many files change :)
Note: you do have to point to a manual storage-initializer image as it won't get built by CI until its checked in.

/assign @yuzisun

File: docs/samples/accelerators/tensorflow-gpu.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   default:
     tensorflow:
-      modelUri: "gs://kfserving-samples/models/tensorflow/flowers"
+      storageUri: "gs://kfserving-samples/models/tensorflow/flowers"
```
Comment (yuzisun) [inline docs/samples/accelerators/tensorflow-gpu.yaml]: These examples are still using v1alpha1, storageUri is in v1alpha2. Let’s just all update them to v1alpha2?

Comment (rakelkar) [reply docs/samples/accelerators/tensorflow-gpu.yaml]: done!

**Comment by yuzisun** [comment]
awesome! we are get closer with v1alpha2 :)

/lgtm
/approve

**Pull request merged**